api/src/main/java/io/opencensus/trace/Span.java

```diff
@@ -259,4 +259,23 @@ public final SpanContext getContext() {
   public final Set<Options> getOptions() {
     return options;
   }
+
+  /**
+   * Type of span. Can be used to specify additional relationships between spans in addition to a
+   * parent/child relationship.
+   *
+   * @since 0.14
+   */
+  public enum Kind {
+    /**
+     * Indicates that the span covers server-side handling of an RPC or other remote network
+     * request.
+     */
+    SERVER,
```

Comment:
/me wants PRODUCER, CONSUMER for christmas, santa

Reply:
We should continue our discussion here https://github.com/census-instrumentation/opencensus-specs/issues/54